src/dlgNotepad.cpp

```diff
@@ -76,3 +84,18 @@ void dlgNotepad::restore()
     notesEdit->setPlainText(txt);
```

Comment:
Does the action of setting the text contents of the widget that `(QPlainTextEdit*) notesEdit` points to cause the generation of the `QPlainTextEdit::textChanged` from that widget? If so, then you may need to wrap the line like this:
```CPP
    notesEdit->blockSignals(true);
    notesEdit->setPlainText(txt);
    notesEdit->blockSignals(false);
```

Reply:
I've tested this PR and it works as expected.

Reply:
Well it is likely to be idempotent as it will just cause the file that has just been loaded to be written back out again but that is a waste to time...

Reply:
Yes you are right :see_no_evil: fixed